src/components/ProductCard.tsx

```diff
@@ -184,18 +184,12 @@ const ProductCard: React.FC<IProps> = ({
 
   const hasRepoData = data && data.repository && !error
 
-  // Check if image is an svg as gatsby-plugin-image doesn't support svg
-  let isSvg = false
-  if (typeof image === "string") {
-    if (image.includes("svg")) {
-      isSvg = true
-    }
-  }
+  const isImgSrc = typeof image === "string"
```

Comment:
If `image` is a string just pass it to the `img` tag since `GatsbyImage` can't process it anyway. 

Furthermore, in all the instances where we are using `ProductCard`, we are not sending a string. Only Gatsby image instances.